samples/cpp/benchmark_app/main.cpp

```diff
@@ -195,6 +195,7 @@ int main(int argc, char* argv[]) {
             slog::info << "GPU extensions is loaded " << ext << slog::endl;
         }
 
+        ov::hint::PerformanceMode ov_perf_hint = ov::hint::PerformanceMode::UNDEFINED;
         if (FLAGS_hint.empty()) {
```

Comment:
On second thought, these changes are not needed, the issue is covered in https://github.com/openvinotoolkit/openvino/pull/10259/
Whole part beneath line 199 (determining default value in case of empty FLAGS_hint should be deleted as well)